rest-api-spec/src/main/resources/rest-api-spec/test/cluster.put_settings/10_basic.yml

```diff
@@ -98,3 +98,116 @@
   - match: {error.root_cause.0.type: "illegal_argument_exception"}
   - match: { error.root_cause.0.reason: "Invalid SearchBackpressureMode: monitor-only" }
   - match: { status: 400 }
+
+
+---
+"Test setting search backpressure cancellation settings":
+  - skip:
+      version: "- 2.99.99"
+      reason: "Fixed in 3.0.0"
```

Comment:
I see backport label added, should this be skipped until 2.17.99 only?

Reply:
After backport would be safer to change it